workspaces/ui-v2/src/entry-points/cloud/cloud-viewer.tsx

```diff
@@ -52,6 +53,13 @@ const appConfig: OpticAppConfig = {
     documentation: {
       allowDescriptionEditing: false,
     },
+    api: {
```

Comment:
What api are we referring to here? Is this the cli api, oauth api or some other api?

Reply:
Good point. The backend (api.useoptic.com). Will clarify. `backendApi`?

Reply:
Perhaps opticApi? We can always change it later fairly easily as since it's only referenced in code